inst/@symfun/diff.m

```diff
@@ -78,7 +78,7 @@
 
 function z = diff(f, varargin)
 
-  z = diff(f.sym, varargin{:});
+  z = diff(f.symbol, varargin{:});
```

Comment:
Since I could not find any direct way of converting a classdef to its parent class, I had to store expr in a `symbol` property. Also, If I name it `sym` then it doesn't work! So I named it `symbol`...